frontend/src/app/components/enterprise/enterprise-trial-waiting/ee-trial-waiting.component.ts

```diff
@@ -51,12 +57,33 @@ export class EETrialWaitingComponent {
   };
 
   constructor(readonly elementRef:ElementRef,
+              readonly cdRef:ChangeDetectorRef,
               readonly I18n:I18nService,
               protected http:HttpClient,
+              readonly Gon:GonService,
               protected notificationsService:NotificationsService,
               public eeTrialService:EnterpriseTrialService) {
   }
 
+  ngOnInit() {
+    let savedDateStr = (this.Gon.get('ee_trial_key') as any).created.split(' ')[0];
+    this.created = new Date(savedDateStr).toLocaleDateString();
```

Comment:
The transformed date has the wrong format. My OpenProject system language is English, the browser language German. However, the date is shown in a German format:
<img width="512" alt="Bildschirmfoto 2020-05-15 um 09 17 38" src="https://user-images.githubusercontent.com/7457313/82022501-10861800-968d-11ea-81d4-c816ac4c7876.png">